src/Functions/formatString.cpp

```diff
@@ -110,7 +110,7 @@ class FormatFunction : public IFunction
                     column->getName(), getName());
         }
 
-        FormatImpl::formatExecute(
+        FormatStringImpl::formatExecute(
```

Comment:
Renamed due to name collision (we already have one `DB::FormatImpl` in `FunctionsConversion.h`)